src/main/kotlin/com/exactpro/th2/lwdataprovider/db/CradleMessageExtractor.kt

```diff
@@ -36,14 +37,15 @@ class CradleMessageExtractor(configuration: Configuration, private val cradleMan
 
     private val storage = cradleManager.storage
     private val batchSize = configuration.batchSize
+    private val codecUsePinAttributes = configuration.codecUsePinAttributes
     
     companion object {
         private val logger = KotlinLogging.logger { }
     }
 
     fun getStreams(): Collection<String> = storage.streams
     
-    fun getMessages(filter: StoredMessageFilter, requestContext: MessageRequestContext) {
+    fun getMessages(filter: StoredMessageFilter, requestContext: MessageRequestContext, responseFormats: List<ResponseFormat>) {
```

Comment:
Let's create our own enum in the LDP to avoid dependencies to the gRPC in the common parts of the LDP